packages/components/select/src/use-select.ts

```diff
@@ -706,9 +724,10 @@ export const useSelectList = () => {
 
   const getListProps: MotionUIPropGetter<"ul"> = useCallback(
     (props = {}, ref = null) => ({
+      id: props.id ?? useId(),
       as: "ul",
       ref: mergeRefs(listRef, ref),
-      role: "select",
+      role: "listbox",
       tabIndex: -1,
       ...props,
```

Comment:
If the `props` has `id`, it is overwritten by the `...props` part.
I think this is fine.
```diff ts
+const id = useId()
 
const getListProps: MotionUIPropGetter<"ul"> = useCallback(
    (props = {}, ref = null) => ({
-      id: props.id ?? useId(),
+      id,
      as: "ul",
      ref: mergeRefs(listRef, ref),
      role: "listbox",
      tabIndex: -1,
      ...props,

```